Content.Server/Construction/ConstructionSystem.Initial.cs

```diff
@@ -203,16 +207,31 @@ void ShutdownContainers()
                             if (used.Contains(entity))
                                 continue;
 
+                            // Dump out any stored entities in used entity
+                            if (TryComp<SharedStorageComponent>(entity, out var storage) && storage.StoredEntities != null)
+                            {
+                                foreach (var storedEntity in storage.StoredEntities.ToList())
+                                {
+                                    var transform = Transform(storedEntity);
+                                    _container.AttachParentToContainerOrGrid(transform);
+                                    _transformSystem.SetLocalPositionRotation(transform, transform.LocalPosition + _random.NextVector2Box() / 2, _random.NextAngle());
+
+                                    storage.Remove(storedEntity);
+                                }
+                            }
+
                             if (string.IsNullOrEmpty(arbitraryStep.Store))
                             {
                                 if (!container.Insert(entity))
                                     continue;
                             }
+
```

Comment:
Were these whitespace changes intended?

Reply:
The white spacing seem fine to me

Reply:
This white space is extraneous because it separates the else from the main body of the if. But don't worry about fixing this, I will fix this for you.